Add tests for lower_graph and fix unpack() ops dispatch

jamesr66a opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #34140 [quantization] Use torchbind for Linear and RNN PackedParams
* **#37540 Add tests for lower_graph and fix unpack() ops dispatch**

Differential Revision: [D21313433](https://our.internmc.facebook.com/intern/diff/D21313433/)

Commit:
Add tests for lower_graph and fix unpack() ops dispatch



Differential Revision: [D21313433](https://our.internmc.facebook.com/intern/diff/D21313433/)

[ghstack-poisoned]

ezyang commented on torch/csrc/jit/passes/onnx/unpack_quantized_weights.cpp:
```diff
@@ -17,6 +17,11 @@ using namespace ::c10::onnx;
 template <class Result, class... Args>
 inline Result callOpUnboxed(const c10::OperatorHandle& op, Args... args) {
```
This function is so weird.

Commit:
Update on "Add tests for lower_graph and fix unpack() ops dispatch"


Differential Revision: [D21313433](https://our.internmc.facebook.com/intern/diff/D21313433/)

[ghstack-poisoned]
